cmd/brew-postgresql-upgrade-database.rb

```diff
@@ -64,8 +64,8 @@
 
   sql_for_lc_collate = "SELECT setting FROM pg_settings WHERE name LIKE 'lc_collate';"
   sql_for_lc_ctype = "SELECT setting FROM pg_settings WHERE name LIKE 'lc_ctype';"
-  lc_collate = Utils.popen_read("#{old_bin}/psql", "postgres", "-qtAc", sql_for_lc_collate).strip
-  lc_ctype = Utils.popen_read("#{old_bin}/psql", "postgres", "-qtAc", sql_for_lc_ctype).strip
+  lc_collate = Utils.popen_read("#{old_bin}/psql", "postgres", "-qtAc", sql_for_lc_collate).gsub(/Time: ([\d.])+ ms/, "").strip
+  lc_ctype = Utils.popen_read("#{old_bin}/psql", "postgres", "-qtAc", sql_for_lc_ctype).gsub(/Time: ([\d.])+ ms/, "").strip
```

Comment:
Is there no way to tell `postgres` to not emit this information or emit it in a parseable format instead e.g. JSON?

Reply:
> Is there no way to tell `postgres` to not emit this information

An alternative would be to run two commands, one to turn off timing:

``` bash
$ psql postgres -qtAc "\timing off" -qtAc "SELECT setting FROM pg_settings WHERE name LIKE 'lc_collate';" 
Timing is off.
en_US.UTF-8
```

Then your setting is always on the second line returned.

There is also apparently a `-X` option to turn off respecting `.psqlrc` but I couldn't get this to work on my local machine (it broke the query altogether). 

> emit it in a parseable format instead e.g. JSON

Nothing listed in the documentation makes me believe this is possible.

Reply:
> There is also apparently a `-X` option to turn off respecting `.psqlrc` but I couldn't get this to work on my local machine (it broke the query altogether).

This seems like the way forward here. Redefining `$HOME` may also help with that?

Reply:
I think it was failing because the `-X` option unsets all of the defaults, so I needed to explicitly pass the username option. 

Here's the command I got working:

```bash
$ psql postgres -qtAX -U $(whoami) -c "SELECT setting FROM pg_settings WHERE name LIKE 'lc_collate';"  
en_US.UTF-8
```

How does that look? Anyway to set the user better in that command?

Reply:
Looks good! Can probably rely on `ENV["USER"]` instead.

Reply:
@MikeMcQuaid Ok, added this is 9a21bd559f9d59d9e264128df6b53f845d4b3222!